web/packages/design/src/DataTable/Table.tsx

```diff
@@ -126,7 +127,15 @@ export function Table<T>({
           </React.Fragment>
         );
       });
-      rows.push(<tr key={rowIdx}>{cells}</tr>);
+      rows.push(
+        <tr
+          key={rowIdx}
+          onClick={row?.onClick ? () => row.onClick(item) : undefined}
```

Comment:
```suggestion
          onClick={() => row.onClick?.(item)}
```

Reply:
i did as you suggested, but it doesn't matter right that it's still technically calling a function (but nothings happening b/c it's undefined)